app/templates/components/paper-menu-content-inner.hbs

```diff
@@ -0,0 +1 @@
+{{yield this}}
```

Comment:
can't you `yield` the `paper-menu-item` from here instead of yielding `this`.
yielding this is considered an anti pattern because we may unecessarily be leaking context.
These are private components, but still, if we can avoid it, I think we should.